extensions/wikia/Wall/WallBaseController.class.php

```diff
@@ -635,21 +622,31 @@ protected function checkAndSetUserBlockedStatus( $wallOwner = null ) {
 
 	}
 
-	public function getThread( $filterid ) {
-		wfProfileIn( __METHOD__ );
-
-		$wallthread = WallThread::newFromId( $filterid );
-		$wallthread->loadIfCached();
 
-		$this->threads = [ $filterid => $wallthread ];
 
-		$this->title = $this->wg->Title;
+	public function message_error() {
 
-		wfProfileOut( __METHOD__ );
 	}
 
-	public function message_error() {
+	/**
+	 * @param Title $title
+	 * @return mixed|string
+	 * @throws MWException
+	 */
+	private function getGreetingText( $title ) {
+		$greeting = Title::newFromText( $title->getText(), NS_USER_WALL_MESSAGE_GREETING );
+		$greetingText = '';
+
+		if ( !empty( $greeting ) && $greeting->exists() ) {
+			$article = new Article( $greeting );
+			$article->getParserOptions();
+			$article->mParserOptions->setIsPreview( true ); // create parser option
+			$article->mParserOptions->setEditSection( false );
+			$greetingText = $article->getParserOutput()->getText();
+		}
 
+		wfRunHooks( 'WallGreetingContent', [ &$greetingText ] );
+		return $greetingText; // used by SWM to add messages to Wall in monobook
 	}
 
 } // end class Wall
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)